pkg/shell/superuser.jsx

```diff
@@ -144,13 +144,13 @@ export class SuperuserDialogs extends React.Component {
         this.superuser = this.superuser_connection.proxy("cockpit.Superuser", "/superuser");
         this.superuser.addEventListener("changed", () => {
             this.setState({
-                show: this.superuser.Current != "root",
+                show: this.superuser.Current != "root" && this.superuser.Current != "",
```

Comment:
So this value can take `"none"`, `"root"`, and what else? Comparing against `!= ""` is not self-explanatory here. Maybe a more standard tristate of null/false/true would be cleaner?

Reply:
Yes, agreed, this isn't clear enough anymore.

The `""` value is used during startup of the bridge until the value either goes to "sudo" or "none".  During that time, the UI should pretend that we have admin access, but the page should not reload if the value goes to "none".

I guess using ´"init"` instead of `""` might make thigns a bit clearer.  I'll have a think about this.

Reply:
>  During that time, the UI should pretend that we have admin access

Should it, actually?  I am starting to get confused...

Reply:
I think it should start with "limited". We used to always enable the privileged operations on the page until sudo detection was complete, and it led/leads to confusion. I had some gut feeling that we fixed that recently. It seems less confusing to start unprivileged until the sudo detection is done (and positive).

Reply:
Ok, I changed this to use `"init"` and documented the values and possible transitions.

Reply:
> I think it should start with "limited". We used to always enable the privileged operations on the page until sudo detection was complete, and it led/leads to confusion. 

Yes, indeed.  It is usually very fast to get admin or to figure out that it wont work, so there is no actual difference in the normal case.  However, sometimes sudo asks three times with a delay in between, or the PAM stack uses something that is slow.

I'll see if I can make the "init" phase fail faster.  We only answer the first prompt anyway, and we can kill the whole process when a second prompt comes.